diff --git a/node_modules/react-native-exif/android/build.gradle b/node_modules/react-native-exif/android/build.gradle
index e1f8079..b9d949d 100644
--- a/node_modules/react-native-exif/android/build.gradle
+++ b/node_modules/react-native-exif/android/build.gradle
@@ -5,7 +5,7 @@ android {
     buildToolsVersion project.hasProperty('buildToolsVersion') ? project.buildToolsVersion : "25.0.3"
 
     defaultConfig {
-        minSdkVersion 16
+        minSdkVersion project.hasProperty('minSdkVersion') ? project.minSdkVersion : 16
         targetSdkVersion project.hasProperty('targetSdkVersion') ? project.targetSdkVersion : 25
         versionCode 1
         versionName "1.0"
