diff --git a/node_modules/react-native-maps/ios/AirMaps/RNMapsMapView.mm b/node_modules/react-native-maps/ios/AirMaps/RNMapsMapView.mm
index 447520d..8043dcb 100644
--- a/node_modules/react-native-maps/ios/AirMaps/RNMapsMapView.mm
+++ b/node_modules/react-native-maps/ios/AirMaps/RNMapsMapView.mm
@@ -60,7 +60,7 @@ - (void)animateToRegion:(NSString *)regionJSON duration:(NSInteger)duration{
     NSDictionary* regionDic = [RCTConvert dictonaryFromString:regionJSON];
     MKCoordinateRegion region = [RCTConvert MKCoordinateRegion:regionDic];
     Boolean animated = duration == 0 ? NO :YES;
-    _view.ignoreRegionChanges = animated;
+    // _view.ignoreRegionChanges = animated;
     [_view setRegion:region animated:animated];
 }
 - (void)setCamera:(NSString *)cameraJSON{
