diff --git a/node_modules/react-native-sensitive-info/build/index.es.js b/node_modules/react-native-sensitive-info/build/index.es.js
index a3b8cf7..2e99837 100644
--- a/node_modules/react-native-sensitive-info/build/index.es.js
+++ b/node_modules/react-native-sensitive-info/build/index.es.js
@@ -1,33 +1,23 @@
 import { NativeModules } from 'react-native';
 
-/*! *****************************************************************************
-Copyright (c) Microsoft Corporation.
+const RNSensitiveInfo = NativeModules.RNSensitiveInfo;
 
-Permission to use, copy, modify, and/or distribute this software for any
-purpose with or without fee is hereby granted.
+RNSensitiveInfo.setInvalidatedByBiometricEnrollment = (
+  invalidatedByBiometricEnrollment,
+) => {
+  if (RNSensitiveInfo.setInvalidatedByBiometricEnrollment == null) {
+    return undefined;
+  }
 
-THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES WITH
-REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF MERCHANTABILITY
-AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY SPECIAL, DIRECT,
-INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES WHATSOEVER RESULTING FROM
-LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR
-OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR
-PERFORMANCE OF THIS SOFTWARE.
-***************************************************************************** */
-
-var __assign = function() {
-    __assign = Object.assign || function __assign(t) {
-        for (var s, i = 1, n = arguments.length; i < n; i++) {
-            s = arguments[i];
-            for (var p in s) if (Object.prototype.hasOwnProperty.call(s, p)) t[p] = s[p];
-        }
-        return t;
-    };
-    return __assign.apply(this, arguments);
+  return RNSensitiveInfo.setInvalidatedByBiometricEnrollment(
+    invalidatedByBiometricEnrollment,
+  );
 };
+RNSensitiveInfo.cancelFingerprintAuth = () => {
+  if (RNSensitiveInfo.cancelFingerprintAuth == null) {
+    return undefined;
+  }
 
-var RNSensitiveInfo = NativeModules.RNSensitiveInfo;
-var RNSensitiveInfo$1 = __assign({}, RNSensitiveInfo);
-
-export default RNSensitiveInfo$1;
-//# sourceMappingURL=index.es.js.map
+  return RNSensitiveInfo.cancelFingerprintAuth();
+};
+export default RNSensitiveInfo;
\ No newline at end of file
diff --git a/node_modules/react-native-sensitive-info/build/index.js b/node_modules/react-native-sensitive-info/build/index.js
index cd860cd..2e99837 100644
--- a/node_modules/react-native-sensitive-info/build/index.js
+++ b/node_modules/react-native-sensitive-info/build/index.js
@@ -1,37 +1,23 @@
-'use strict';
+import { NativeModules } from 'react-native';
 
-Object.defineProperty(exports, '__esModule', { value: true });
+const RNSensitiveInfo = NativeModules.RNSensitiveInfo;
 
-var reactNative = require('react-native');
+RNSensitiveInfo.setInvalidatedByBiometricEnrollment = (
+  invalidatedByBiometricEnrollment,
+) => {
+  if (RNSensitiveInfo.setInvalidatedByBiometricEnrollment == null) {
+    return undefined;
+  }
 
-/*! *****************************************************************************
-Copyright (c) Microsoft Corporation.
-
-Permission to use, copy, modify, and/or distribute this software for any
-purpose with or without fee is hereby granted.
-
-THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES WITH
-REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF MERCHANTABILITY
-AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY SPECIAL, DIRECT,
-INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES WHATSOEVER RESULTING FROM
-LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR
-OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR
-PERFORMANCE OF THIS SOFTWARE.
-***************************************************************************** */
-
-var __assign = function() {
-    __assign = Object.assign || function __assign(t) {
-        for (var s, i = 1, n = arguments.length; i < n; i++) {
-            s = arguments[i];
-            for (var p in s) if (Object.prototype.hasOwnProperty.call(s, p)) t[p] = s[p];
-        }
-        return t;
-    };
-    return __assign.apply(this, arguments);
+  return RNSensitiveInfo.setInvalidatedByBiometricEnrollment(
+    invalidatedByBiometricEnrollment,
+  );
 };
+RNSensitiveInfo.cancelFingerprintAuth = () => {
+  if (RNSensitiveInfo.cancelFingerprintAuth == null) {
+    return undefined;
+  }
 
-var RNSensitiveInfo = reactNative.NativeModules.RNSensitiveInfo;
-var RNSensitiveInfo$1 = __assign({}, RNSensitiveInfo);
-
-exports.default = RNSensitiveInfo$1;
-//# sourceMappingURL=index.js.map
+  return RNSensitiveInfo.cancelFingerprintAuth();
+};
+export default RNSensitiveInfo;
\ No newline at end of file
