diff --git a/node_modules/react-native-volume-manager/android/build.gradle b/node_modules/react-native-volume-manager/android/build.gradle
index ecc8e04..a8d2b17 100644
--- a/node_modules/react-native-volume-manager/android/build.gradle
+++ b/node_modules/react-native-volume-manager/android/build.gradle
@@ -28,7 +28,7 @@ def getExtOrIntegerDefault(name) {
 android {
   compileSdkVersion getExtOrIntegerDefault('compileSdkVersion')
   defaultConfig {
-    minSdkVersion 16
+    minSdkVersion rootProject.hasProperty('minSdkVersion') ? rootProject.minSdkVersion : 16
     targetSdkVersion getExtOrIntegerDefault('targetSdkVersion')
     versionCode 1
     versionName "1.0"

